src/fetch-mock.js

```diff
@@ -269,7 +271,7 @@ FetchMock.setGlobals = function (globals) {
 	Object.assign(FetchMock, globals)
 }
 
-FetchMock.prototype.sandbox = function () {
+FetchMock.prototype.sandbox = function (promise) {
```

Comment:
I'd call it `Promise` with first capital to emphasize it's a constructor, not an instance (and same everywhere where it's passed around).